src/modules/uiKit/Button/useButtonStyles.ts

```diff
@@ -9,6 +9,9 @@ export const useButtonStyles = makeStyles<Theme>(theme => ({
 
   rounded: {
     borderRadius: 44,
+    '&:before': {
+      borderRadius: 44,
```

Comment:
```suggestion
      borderRadius: 'inherit',
```
I think this variant would be a bit better 🙂